runtime/centrifuge/src/lib.rs

```diff
@@ -1665,6 +1665,7 @@ pub type Executive = frame_executive::Executive<
 		pallet_preimage::migration::v1::Migration<Runtime>,
 		pallet_democracy::migrations::v1::Migration<Runtime>,
```

Comment:
Do we still need those?

Reply:
According to #1286, we don't. So let's remove them.

Reply:
Could they be included in `altair` but not in `centrifuge`? In that case, we need them here. In `altair` I checked a local migration with those, and I get an error because they were already migrated, doing the same thing for centrifuge would give the same error

Reply:
I think this command should tell that:
```sh
RUST_LOG=runtime=trace,try-runtime::cli=trace,executor=trace cargo run --release --features try-runtime try-runtime --execution Native --chain altair-local --no-spec-check-panic on-runtime-upgrade live --uri wss://fullnode.parachain.centrifuge.io:443
````

Reply:
Removed.